cirq/circuits/circuit_test.py

```diff
@@ -1673,6 +1673,25 @@ def test_with_parameters_resolved_by():
       │
 1: ───@^0.1───Y^0.2───
 """)
+    q = cirq.NamedQubit('q')
+    # no-op parameter resolution
+    circuit = cirq.Circuit([
+        cirq.Moment(), cirq.Moment([cirq.X(q)])])
+    resolved_circuit = cirq.resolve_parameters(
+        circuit,
+        cirq.ParamResolver({}))
+    assert circuit == resolved_circuit
```

Comment:
Use `cirq.testing.assert_same_circuit`, just because it has significantly better error output.